docs/ROADMAP.md

```diff
@@ -15,13 +15,13 @@
 - Asset loading
 - glTF support 
 - Testing utilities
+- Windows, macOS, Linux compatibility
 
 ## Planned Features
-
-- Windows, macOS, Linux compatibility (in that order)
 - A scripting language
-- Basic UI library
+- Fully featured and extensible UI Library.
 - An editor
+- Physics
```

Comment:
Should this be expanded (because of the "Basic physics" above)

Reply:
ah right, we don't have physics :P